### PR TITLE
osin: bump golang to 1.15

### PR DESCRIPTION
**Pull request by stlaz**

### CODE REVIEW

**Comment by stlaz** [comment]
not sure if nobody cares about centos 8 or it just uses a different tag, moving back to 7

**Comment by stlaz** [comment]
hopefully 1.15 won't attempt to use vendor/ anymore as in https://github.com/openshift/osin/pull/200?

File: ci-operator/config/openshift/osin/openshift-osin-master.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ resources:
     requests:
       cpu: "3"
       memory: 8Gi
-test_binary_build_commands: go build -race ./...
+test_binary_build_commands: go build -race -mod=mod ./...
```
Comment (tnozicka) [inline ci-operator/config/openshift/osin/openshift-osin-master.yaml:20]: openshift builds require offline builds and vendor folder. openshift/osin needs a bump commit

**Review by tnozicka — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/osin/openshift-osin-master.yaml (hunk 1 above)
Comment (stlaz) [reply ci-operator/config/openshift/osin/openshift-osin-master.yaml:20]: This repository is not being built offline, it has never had a vendor/ folder. I am merely using openshift CI for the community contributions to it.

**Comment by stlaz** [comment]
/retest
maybe the changes finally got picked up?

**Review by tnozicka — APPROVED** [review]
/lgtm

**Pull request merged**